docs/getting-started-guides/kubeadm.md

```diff
@@ -13,6 +13,8 @@ The installation uses a tool called `kubeadm` which is part of Kubernetes 1.4.
 This process works with local VMs, physical servers and/or cloud servers.
 It is simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
 
+The full reference for `kubeadm` is [here](/docs/admin/kubeadm/).  Please refer to that for all of the command line flags and advice for automating `kubeadm` itself.
```

Comment:
There are two links on this page to the reference. 
If you're convinced that we need a third, it's fine for me, otherwise move or remove